specification/patient-demographics-service-api.yaml

```diff
@@ -139,56 +139,69 @@ paths:
         url: https://digital.nhs.uk/services/organisation-data-service
       description: |
         ## Overview
-        Use this endpoint to search for a patient record in PDS.
+        Use this endpoint to search for a patient in PDS.
```

Comment:
We might want to remove references to PDS -- product owner thing I guess but want to flag for future.